DevelopGitHub/GitHubExtension/GitHubExtension.Infrastructure.ActionFilters/Extensions/ActivityFiltersExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Web.Http.Dependencies;
+using System.Web.Http.Filters;
+using GitHubExtension.Infrastructure.ActionFilters.Constants;
+using GitHubExtension.Infrastructure.ActionFilters.Models;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Infrastructure.ActionFilters.Extensions
+{
+    public static class ActivityFiltersExtensions
+    {
+        public static IDependencyResolver GetDependencyResolver(this HttpActionExecutedContext actionExecutedContext)
+        {
+            return actionExecutedContext.ActionContext.RequestContext.Configuration.DependencyResolver;
```

Comment:
when injecting dependencies into Web API filter attributes, get my instances from dependency resolver http://simpleinjector.readthedocs.io/en/latest/webapiintegration.html , according to this documentation. For example:
var checker = GlobalConfiguration.Configuration.DependencyResolver
            .GetService(typeof(IMinimumAgeChecker)) as IMinimumAgeChecker;